cmd/relay-gossip/main.go

```diff
@@ -32,8 +32,9 @@ func main() {
 		Usage:   "pubsub relay for randomness beacon",
 		Flags: []cli.Flag{
 			&cli.StringFlag{
-				Name:    "network-name",
-				Aliases: []string{"nn"},
+				Name:     "chain-hash",
+				Aliases:  []string{"h"},
+				Required: true,
```

Comment:
can't the relay derive this automatically from the drand group it's been connected to if not supplied?

Reply:
Yes, true, it doesn't need to be required - will fix.

Reply:
Ok so, it needs to be required right now so that the relay will publish to the correct topic.

It's a bigger change to look it up from the gRPC API and tricky because currently the topic is joined in the constructor and the gRPC client is created in `start`.

Reply:
defer to an issue then.
the re-factor to support it will help this both be more user friendly and make it easier to transition to a cdn client rather than grpc

Reply:
👍 I'll create one as soon as I get a sec